webpack.config.js

```diff
@@ -408,6 +409,7 @@ function* webpackConfig( env, argv ) {
 	yield {
 		entry: {
 			'googlesitekit-i18n': './assets/js/googlesitekit-i18n.js',
+			'googlesitekit-element': './assets/js/googlesitekit-element.js',
```

Comment:
As mentioned previously, this should be a "Basic Module" below rather than grouped with our internal modules. In order for it to be usable as an external library/dependency, it needs to be decoupled from our entry points.

Reply:
My mistake, this part is correct :)